Users/Docker_Image.rst

```diff
@@ -11,6 +11,16 @@ on your code in the working directory, like this:
 
     docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
 
+In some computer operating systems and programming languages,
+an alias is an alternative and usually easier-to-understand or 
+more significant name for a defined data object.
```

Comment:
Please specify which shell alias syntax you are using here(I think it's bash syntax).